src/auro-badge.js

```diff
@@ -24,18 +25,35 @@ import styleCss from "./style-css.js";
 
 // build the component class
 class AuroBadge extends LitElement {
-  // constructor() {
-  //   super();
-  // }
+  constructor() {
+    super();
+    this.dom = new DOMParser().parseFromString(closeIcon.svg, 'text/html');
+    this.svg = this.dom.body.firstChild;
+  }
 
   // function to define props used within the scope of this component
   static get properties() {
     return {
       // ...super.properties,
-      cssClass:   { type: String }
+      action: {type: Boolean},
```

Comment:
I feel that the name `action` doesn't imply what you want. I would suggest using `target` or something related to the fact that this is creating a clickable event target.